docs/WiaAndRvz.md

```diff
@@ -245,3 +245,23 @@ u32* buffer_ptr;
 
 buffer_ptr++;
 ```
+
+## Block Sizes
+
+When archiving to a disc image, it is important to choose block sizes wisely.
+
+### Requirements
+
+- GCZ: Block size "must" be a power of 2 (It has not been verified that things break if it isn't! But using a power of 2 is at the very least recommended)
+
+- WIA: Block size must not be smaller than `2 MiB`. Block size must be an integer multiple of `2 MiB`.
+
+- RVZ: Block size must not be smaller than `32 KiB`. Block sizes smaller than `2 MiB` must be a power of 2. Block sizes larger than `2 MiB` must be an integer multiple of `2 MiB`.
+
+### Recommendations
+
+- Block sizes smaller than `32 KiB` are not recommended (performance doesn't increase anymore after that point)
```

Comment:
This isn't useful to note for RVZ and WIA, since they don't support sizes smaller than that anyway.